src/logic/exceptions/registry.ts

```diff
@@ -30,6 +30,7 @@ enum ErrorCodes {
   _616 = '616: Failed to retrieve recommended nonce',
   _617 = '617: Error fetching safeTxGas',
   _618 = '618: Error fetching fee history',
+  _619 = '619: Failed prepairing a multisend tx',
```

Comment:
Small typo here + all other texts starting with "Failed" use "Failed to ..."
```suggestion
  _619 = '619: Failed to prepare a multisend tx',
```